ledger/heavy/executor/jetkeeper_test.go

```diff
@@ -34,6 +34,12 @@ import (
 	"github.com/insolar/insolar/instrumentation/inslogger"
 )
 
+func Test_JetKeeperKey(t *testing.T) {
```

Comment:
Not necessary.

Reply:
I don't agree

Reply:
Agreed to create new file for white-box testing.